Compile error in status_server on Mac OS

## Bug Report

<!-- Thanks for your bug report! Don't worry if you can't fill out all the sections. -->

### What version of TiKV are you using?
<!-- You can run `tikv-server --version` -->

Master
### What operating system and CPU are you using?
<!-- If you're using Linux, you can run `cat /proc/cpuinfo` -->

### Steps to reproduce
<!-- If possible, provide a recipe for reproducing the error. A complete runnable program is good. -->

run 
### What did you expect?

Everything goes OK

### What did happened?
```
error[E0061]: this function takes 6 arguments but 5 arguments were supplied
    --> src/server/status_server/mod.rs:1255:33
     |
1255 |         let mut status_server = StatusServer::new(
     |                                 ^^^^^^^^^^^^^^^^^ expected 6 arguments
1256 |             1,
     |             -
1257 |             None,
     |             ----
1258 |             ConfigController::default(),
     |             ---------------------------
1259 |             Arc::new(SecurityConfig::default()),
     |             -----------------------------------
1260 |             MockRouter,
     |             ---------- supplied 5 arguments
     |
note: associated function defined here
    --> src/server/status_server/mod.rs:94:12
     |
94   |     pub fn new(
     |            ^^^
95   |         status_thread_pool_size: usize,
     |         ------------------------------
96   |         pd_client: Option<Arc<RpcClient>>,
     |         ---------------------------------
97   |         cfg_controller: ConfigController,
     |         --------------------------------
98   |         security_config: Arc<SecurityConfig>,
     |         ------------------------------------
99   |         router: R,
     |         ---------
100  |         store_path: PathBuf,
     |         -------------------
```

This problem seems to be introduced by #11266

--- a/src/server/status_server/mod.rs
+++ b/src/server/status_server/mod.rs
@@ -1258,7 +1258,7 @@ mod tests {
             ConfigController::default(),
             Arc::new(SecurityConfig::default()),
             MockRouter,
-            std::env::temp_dir()
+            std::env::temp_dir(),
         )
         .unwrap();
         let addr = "127.0.0.1:0".to_owned();

--- a/src/server/status_server/mod.rs
+++ b/src/server/status_server/mod.rs
@@ -1258,6 +1258,7 @@ mod tests {
             ConfigController::default(),
             Arc::new(SecurityConfig::default()),
             MockRouter,
+            std::env::temp_dir()
         )
         .unwrap();
         let addr = "127.0.0.1:0".to_owned();

